README.md

```diff
@@ -1,235 +1,436 @@
-# Kaspa on Rust
-
-This repository contains the implementation of the Kaspa full-node and related libraries in the Rust programming language. This is an Alpha version at the initial testing phase, however the node is expected to be fully functional and capable as a drop-in replacement for the Kaspa golang node.
-
-## Getting started
-
-- General prerequisites:
-  - Linux: `sudo apt install build-essential libssl-dev pkg-config`
-  - Windows: [Git for Windows](https://gitforwindows.org/) or an alternative Git distribution.
-- Install Protobuf (required for gRPC)
-  - Linux: `sudo apt install protobuf-compiler libprotobuf-dev`
-  - Windows: [protoc-21.10-win64.zip](https://github.com/protocolbuffers/protobuf/releases/download/v21.10/protoc-21.10-win64.zip) and add `bin` directory to `Path`
-  - MacOS: `brew install protobuf`
-- Install the [clang toolchain](https://clang.llvm.org/) (required for RocksDB and WASM `secp256k1` builds)
-  - Linux: `apt-get install clang-format clang-tidy clang-tools clang clangd libc++-dev libc++1 libc++abi-dev libc++abi1 libclang-dev libclang1 liblldb-dev libllvm-ocaml-dev libomp-dev libomp5 lld lldb llvm-dev llvm-runtime llvm python3-clang`
-  - Windows: Please see [Installing clang toolchain on Windows](#installing-clang-toolchain-on-windows)
-  - MacOS: Please see [Installing clang toolchain on MacOS](#installing-clang-toolchain-on-macos)
-- Install the [rust toolchain](https://rustup.rs/)
-- If you already have rust installed, update it by running: `rustup update`
-- Install wasm-pack: `cargo install wasm-pack`
-- Install wasm32 target: `rustup target add wasm32-unknown-unknown`
-- Run the following commands:
 
-```bash
-$ git clone https://github.com/kaspanet/rusty-kaspa
-$ cd rusty-kaspa
-```
+<h1 align="center">Kaspa On Rust</h1>
+
+Welcome to the Rust-based implementation of the Kaspa full-node and its ancillary libraries. This Alpha release serves as a replacement to the established <a href="https://github.com/kaspanet/kaspad">Golang node</a> (once the rust rewrite is completed), introducing developers to the possibilities of Rust in the Kaspa network's context.
+
+Our codebase is an experimental ground for developers aiming to push the boundaries of blockchain scalability and speed. The Kaspa node aims to validate the feasibility of a high-speed blockchain, boasting block intervals of one second with prospects for significant enhancements.

```

Comment:
I think paragraph 2 (Our codebase...) and paragraph 3 (Contributors to this repo...) currently don't add too much value and should be removed. Ideally, the introduction section is small enough so we're not immediately seeing a wall of text.

In a different PR, after this is merged, We can add a `#Contribution` section to the bottom of this doc that outlines how to contribute code or non-code artifacts.